jex/src/jexir/from-json-schema.test.ts

```diff
@@ -2,67 +2,61 @@ import { JSONSchema7 } from 'json-schema'
 import { test, expect } from 'vitest'
 import { JexIR } from './typings'
 import { fromJsonSchema as j2x } from './from-json-schema'
-import { jexEquals as jexEquals } from '../jex-equals'
```

Comment:
**note:** I wanted to make this test suite more unit or isolated. So I removed usage of `jexEquals` which is tested on its own